requirements.txt

```diff
@@ -5,13 +5,16 @@ kiwisolver==1.1.0
 matplotlib==3.1.2
 numpy==1.18.1
 pandas==0.25.3
-pkg-resources==0.0.0
 pyparsing==2.4.6
 python-dateutil==2.8.1
 pytz==2019.3
-scikit-learn==0.22.1
+scikit-learn==0.22.2.post1
 scipy==1.4.1
-sent2vec-prebuilt==0.0.3
 six==1.14.0
 sklearn==0.0
-sentence-transformers<=1.2.3
+sentence-transformers<=0.2.6.1
+sent2vec-prebuilt==0.0.3
+mpi4py==3.0.3
+transformers==2.11.0
```

Comment:
Where is this used?

Reply:
Transformers is used in the Sentence-Transformers model. I called it specifically here, because for some reason it did not have the right version until I did that.

Reply:
On `master`, `sentence-transformers` has version `1.2.3`. I haven't run into any issues with needing to include `transformers` in the couple times I have used it. Is there a reason this PR uses version `0.2.6.1`?
https://github.com/byu-dml/d3m-profiler/blob/9a3bc45061267091b0109f2159648785e370a18b/requirements.txt#L17

Reply:
I figured out that the latest version of `sentence-transformers` is `0.2.6.2`, which came out a week ago (not sure why version is set to `1.2.3` on `master`). Seems like this isn't an issue then